ext/mbstring/mbstring.h

```diff
@@ -53,6 +53,10 @@ PHP_RINIT_FUNCTION(mbstring);
 PHP_RSHUTDOWN_FUNCTION(mbstring);
 PHP_MINFO_FUNCTION(mbstring);
 
+/* When converting encoded text to a buffer of wchars (Unicode codepoints),
+ * the buffer must be at least this size */
+#define MBSTRING_MIN_WCHAR_BUFSIZE 5
```

Comment:
This should probably be in libmbfl, next to the definition of the to_wchar prototype.

Reply:
OK, that makes sense.

Thanks very much.